falcon_kit/run_support.py

```diff
@@ -237,6 +237,14 @@ def get_dict_from_old_falcon_cfg(config):
         else:
             falcon_sense_skip_contained = False
 
+    falcon_greedy = False
+    if config.has_option(section, 'falcon_greedy'):
+        falcon_greedy = config.get(section, 'falcon_greedy')
```

Comment:
falcon_greedy = config.getboolean(section, 'falcon_sense_greedy')

> A convenience method which coerces the option in the specified section to a Boolean value. Note that the accepted values for the option are "1", "yes", "true", and "on", which cause this method to return True, and "0", "no", "false", and "off", which cause it to return False. These string values are checked in a case-insensitive manner. Any other value will cause it to raise ValueError.

Yeah, I see that you're just copying the pattern for falcon_sense_skip_contained, but I've used `configboolean()` elsewhere. You can change skip_contained here too, since this whole change is very small.